README.md

```diff
@@ -83,3 +83,14 @@ the following in your catalina.out log when you press "Launch" (jsfiddle) or
 
    MyRequest: true
 
+[Docker based compilation]
```

Comment:
nice docker support!

Reply:
Yeah, I *really* like compiling java projects via docker. Avoids having to install anything locally, allows anyone to run it and keeps the versions fixed. Could probably put this as a shell script locally too but easy to copy/paste for the moment.